internal/v1_1/generator.go

```diff
@@ -46,10 +47,17 @@ func NewTemplateGenerator(contractInfos ContractInfos, logger output.Logger, net
 		if err != nil {
 			return nil, fmt.Errorf("could not create grpc gateway for %s %w", network.Name, err)
 		}
-		state, err := flowkit.Init(loader, crypto.ECDSA_P256, crypto.SHA3_256)
+		state, err := flowkit.Init(loader)
 		if err != nil {
 			return nil, fmt.Errorf("could not initialize flowkit state %w", err)
 		}
+
+		emulatorAccount, err := accounts.NewEmulatorAccount(loader, crypto.ECDSA_P256, crypto.SHA3_256, "")
+		if err != nil {
+			return nil, fmt.Errorf("could not create emulator account %w", err)
+		}
+		state.Accounts().AddOrUpdate(emulatorAccount)
```

Comment:
curious if flixkit even needs the emulator-account for this

Reply:
Good point. Removed.